src/axom/quest/docs/sphinx/read_mesh.rst

```diff
@@ -50,3 +57,48 @@ The following example shows usage of the STLReader class:
 After reading the STL file, the ``STLReader::getMesh`` method gives access to the
 underlying mesh data.  The reader may then be deleted.
 
+Reading a Pro/E file
+--------------------
+
+As read by Axom, a Pro/E tet file contains:
+
+- Zero or more comment lines starting with a ``#`` character
+- One line with two integers: the number of nodes n and the number of tetrahedra t
+- n lines, one for each node; each line contains a contiguous integer ID starting at 1
+  and three floating-point literals specifying the node location
+- t lines, one for each tetrahedron; each line contains a contiguous integer ID
+  starting at 1 and four integers specifying the tet's nodes
+
+Reading an ASCII Pro/E tet file is similar to reading an STL file.  The code
+examples are excerpts from the file ``<axom>/src/axom/quest/examples/quest_proe_bbox.cpp``.
+
+We include the ProEReader header
+
+.. literalinclude:: ../../examples/quest_proe_bbox.cpp
+   :start-after: _read_proe_include1_start
+   :end-before: _read_proe_include1_end
+   :language: C++
+
+and also the mint Mesh and UnstructuredMesh headers.
+
+.. literalinclude:: ../../examples/quest_proe_bbox.cpp
+   :start-after: _read_proe_include2_start
+   :end-before: _read_proe_include2_end
+   :language: C++
+
+For convenience, we specify some type aliases.
+
+.. literalinclude:: ../../examples/quest_proe_bbox.cpp
+   :start-after: _read_proe_typealiases_start
+   :end-before: _read_proe_typealiases_end
+   :language: C++
+
+The following example shows how to use the ProEReader class.
+
+.. literalinclude:: ../../examples/quest_proe_bbox.cpp
+   :start-after: _read_proe_file_start
+   :end-before: _read_proe_file_end
+   :language: C++
+
+After reading the Pro/E file, the ``ProEReader::getMesh`` method gives access to the
+underlying mesh data.  The reader may then be deleted.
```

Comment:
This is really nice!

It might also be good to mention that the ProE reader has the ability to load a subset of the mesh tetrahedra based on a predicate defined in terms of a provided BoundingBox.

Reply:
I mentioned it in the code, in a comment.  I'll also mention it in the Sphinx text.